pyiode/pyiode_exec.pyx

```diff
@@ -6,18 +6,109 @@
+            
+    var_files: Optional[Union[str, List[str]]] = None
+            list of files the needed variables must be read from. "WS" means current current Variable WS.
+            if empty, only the current KV_WS is used
+            
+    scl_files: Optional[Union[str, List[str]]] = None
+            list of files the needed scalars must be read from. "WS" means current current WS.
+            if empty, only the current workspace is used
+                        
+    trace: int = 0
+            optional trace indicating the source of the variables and scalars. For example:
+            
+            Execution of identities
+            
+            Parameters
+                Execution sample : 1960Y1:2015Y1
+            
+                Variables loaded
+                    From WS : AOUC AOUC_ COTRES DEBT DTF DTH EFMY EFXY EX FLG FLGR GAP2 GAP_ GOSF HF  
+                    
+                Scalars loaded
+                    From ../data/fun : gamma gamma2 gamma3 gamma4 knf3
+            
+    Examples
+    --------
+        # define the "trace" file
+        iode.w_dest("test_iode.htm", iode.W_HTML)
+    
+        # Simple case: no vars in WS, sample given
+        iode.ws_clear_var()
+        iode.ws_sample_set("1960Y1", "2015Y1")
+        iode.ws_load_idt(f"{IODE_DATA_DIR}fun")
+        iode.idt_execute("1961Y1:2000Y1", "AOUC", f"{IODE_DATA_DIR}fun", trace = 1)
+        AOUC = iode.get_var("AOUC")
+        test_eq('iode.idt_execute("1961Y1:2000Y1", "AOUC", f"{IODE_DATA_DIR}fun, trace = 1") -> AOUC[1961Y1]', 0.24783192, AOUC[1])
+    
+        # Load WS + change value before execution to check the result
+        iode.ws_load_var(f"{IODE_DATA_DIR}fun")
+        AOUC = iode.get_var("AOUC")
+        AOUC[1] = 0.1 
+        iode.set_var("AOUC", AOUC) # var changed in IODE WS
+        iode.idt_execute("1961Y1:2015Y1", "AOUC")
+        AOUC = iode.get_var("AOUC")
+        test_eq('2. iode.idt_execute("1961Y1:2015Y1", "AOUC") -> AOUC[1961Y1]', 0.24783192, AOUC[1])
+    
+        # Shortest way using get_var_as_ndarray(..., False) => no iode.set_var() needed
+        iode.ws_load_var(f"{IODE_DATA_DIR}fun")
+        AOUC = iode.get_var_as_ndarray("AOUC", False)
+        AOUC[1] = 0.1
+        iode.idt_execute("1961Y1:2015Y1", "AOUC")
+        test_eq('3. iode.idt_execute("1961Y1:2015Y1", "AOUC") -> AOUC[1961Y1]', 0.24783192, AOUC[1])
+    
+        # Alternative ways to call idt_execute
+        AOUC = iode.get_var_as_ndarray("AOUC", False)
+        AOUC[1] = 0.1
+        iode.idt_execute(["1961Y1", "2015Y1"], ["AOUC", "FLGR"])
+        test_eq('4. iode.idt_execute(["1961Y1", "2015Y1"], ["AOUC", "FLGR"]) -> AOUC[1961Y1]', 0.24783192, AOUC[1])
+    
+        # Call with empty parameters 
+        iode.ws_load_scl(f"{IODE_DATA_DIR}fun")
+        iode.delete_idt("SSFFX") # SSFFX contains non existent variables
+        AOUC = iode.get_var_as_ndarray("AOUC", False)
+        print(f"AOUC[1]={AOUC[1]}")
```

Comment:
Avoid to use print in doctests.
Instead rewrite "print()" by Python comments `#`

Here:
```python
>>> # AOUC[1] = 0.24783192
```

Reply:
Added TODO in source code